scripts/make_bdry_files.py

```diff
@@ -52,34 +52,37 @@
 src_grd = CGrid_glorys.A2CGrid(fpath, xrange=xrange, yrange=yrange, lonoffset=0, **varstr)
 dst_grd = pyroms.grid.get_ROMS_grid(gridname)  # read data from the gridid.txt
 
-for it, tfile in zip(timerange, file):
-    itstr = str(it).replace(' ', 'T')
-
-    # -- interpolates vector fields -- #
-    dfileu = f'{dst_grd.name}_u_bdry.nc'  # temporary interpolated file name
-    dfilev = f'{dst_grd.name}_v_bdry.nc'  # temporary interpolated file name
-
-    # interpolating
-    remap_bdry.make_bdry_uv_file(tfile, src_grd, dst_grd, itstr,
-                   dst_fileu=dfileu, dst_filev=dfilev, wts_file=wfiles)
-
-    # -- interpolates scalar fields -- #
-    for varb in ['so', 'thetao','zos']:
-        dfile = f'{dst_grd.name}_{varb}_bdry.nc'  # temporary interpolated file
-        remap_bdry.make_bdry_rho_file(tfile, src_grd, dst_grd, varb, itstr,
-                              dst_file=dfile, weight_file=wfiles[0])
-
-    # name of the merged initial condition file
-    bdry_file = osp.join(outdir, f'{dst_grd.name}_bdry_{itstr}.nc')
-    out_file = f'{dst_grd.name}_%s_bdry.nc'  # temporary interpolated file
-
-    # -- merge files and remove temporary files --#
-    cont = 0
-    for istr in ['zos', 'so', 'thetao', 'u', 'v']:
-        if cont ==0:
-            command = ('ncks', '-a', '-O', out_file % istr, bdry_file)  # create ic
-            cont += 1
-        else:
-            command = ('ncks', '-a', '-A', out_file % istr, bdry_file)  # append
-        subprocess.call(command)
-        os.remove(out_file % istr)  # remove temp files
+# using only one file to create an empty boundary grid template, which will be filled with
+# global outputs by the scripts_xesmf/make_bdry_file_replace.py
+it,tfile = timerange[0], file[0]
```

Comment:
I remember that the it and tfile should be lists, we need to test this before merging the commit

Reply:
This change is working correctly